package.json

```diff
@@ -39,8 +43,8 @@
         "big.js": "5.2.2",
         "chalk": "2.4.2",
         "ethers": "5.1.3",
-        "luxon": "1.26.0",
         "js-yaml": "4.1.0",
+        "luxon": "^1.27.0",
```

Comment:
it would be better if use `npm --save-dev --save-exact` to add the package

Reply:
fixed.